doc/developer-guides/hld/virtio-net.rst

```diff
@@ -533,7 +533,7 @@ a PCI slot to the device model acrn-dm as shown below.
 
 .. code-block:: none
 
-   -s 4,virtio-net,<macvtap_name>,[mac=<XX:XX:XX:XX:XX:XX>]
+   -s 4,virtio-net,<macvtap=name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Comment:
```suggestion
   -s 4,virtio-net,macvtap=<name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Reply:
Is `macvtap` a valid name by the way, I did not think so so perhaps the right line should rather be:
```suggestion
   -s 4,virtio-net,tap=<name>,[mac=<XX:XX:XX:XX:XX:XX>]
```
?